packages/cyberstorm/src/components/MetaItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/MetaItem.module.css";
+
+export interface MetaItemProps {
+  label?: string;
+  icon?: ReactNode;
+  metaItemStyle?: string;
+}
+
+/**
+ * Cyberstorm MetaItem component
```

Comment:
"MetaItem" might not tell people that much? Maybe describe what the component looks like or where it's supposed to be used?

Reply:
Could be anything, "InformationDisplay" "DataPoint" etc... Don't want to use too much time to think about a better name, we'll see if a better one comes up. Shouldn't be _too bad_ of a refactor to find+replace metaitem with something else.

Reply:
Well I didn't really mean to suggest renaming it, rather to use the comment field to describe what MetaItems are used for. Not a big deal, but currently I kinda know that this is a React component named MetaItem that belongs to Cyberstorm package even without the comment, so it doesn't add that much value.

Reply:
Now has a bit more descriptive description